### ISSUE TITLE
be able to exclude job from automatic recreation

### ISSUE THREAD

**Issue by vdelendik** [qaprosoft/qps-pipeline#17]
Sometimes we test/debug new features on CI and update job manually adding extra arguments, changing existing one etc.
But as only push happens scanner recreate this job forcibly and we have to declare new fields again.

It would be nice to have possibility to avoid job re-creation via scanner.
Proposed solution:
1) reuse description field for that putting into it some keyword like:
DO NOT RECREATE, SCANNER-OUT etc :)
2) scanner job before starting re-creation should just analyze description content and do not force re-creation until this description exists
3) but to be informed that some jobs are not in sync with latest jobdsl/pipeline it is recommended t inform suite owners and swat admin team about it

**Comment by tsvrko** [comment]
will be available in version 2.7+ via parameter ignore_existing
